ui/test/utils/timeSeriesToDygraph.test.js

```diff
@@ -1,6 +1,6 @@
 import timeSeriesToDygraph, {
   timeSeriesToTableGraph,
-} from 'src/utils/timeSeriesToDygraph'
+} from 'src/utils/timeSeriesTransformers'
```

Comment:
This file should be renamed timeSeriesTransformers since that is what it is testing now. Also it should test all the functions in timeSeriesTransformers as mentioned above.